src/sinks/elasticsearch.rs

```diff
@@ -128,9 +131,10 @@ impl ElasticSearchCommon {
             format!("Basic {}", base64::encode(token.as_bytes()))
         });
 
-        let region: Option<Region> = match config.region {
-            Some(ref region) => Some(region.try_into()?),
-            None => None,
+        let region: Option<Region> = match config.region.clone().try_into() {
```

Comment:
It's a little ugly, but this can be `(&config.region).try_into()` to avoid the clone.